src/components/MyKiva/LendingStats.vue

```diff
@@ -28,26 +28,28 @@
 				</span>
 				<div v-if="loanRegions" class="tw-flex tw-flex-col md:tw-flex-row tw-gap-y-2 md:tw-gap-x-6">
 					<ul
-						class="tw-grid tw-grid-cols-2 sm:tw-grid-cols-3 md:tw-grid-cols-4 tw-gap-y-2 tw-gap-x-2
+						class="tw-grid tw-grid-cols-2 sm:tw-grid-cols-3 md:tw-grid-cols-4 tw-gap-y-1 tw-gap-x-0.5
 							tw-w-full"
 					>
 						<li
 							v-for="(region, idx) in props.regionsData"
 							:key="region.name"
 							class="tw-flex tw-items-center tw-min-w-0 tw-overflow-hidden tw-w-full"
 						>
-							<RoundCheckbox
+							<kv-checkbox
```

Comment:
Once the other PR is complete, you'll also need to install the latest npm package version.